docs/utilities/idempotency.md

```diff
@@ -151,18 +151,20 @@ Depending on your version of Java (either Java 1.8 or 11+), the configuration sl
         }
         
         dependencies {
-            aspect 'software.amazon.lambda:powertools-idempotency:{{ powertools.version }}'
+            aspect 'software.amazon.lambda:powertools-idempotency-dynamodb:{{ powertools.version }}'
         }
         
         sourceCompatibility = 1.8
         targetCompatibility = 1.8
     ```
 
 ### Required resources
-
 Before getting started, you need to create a persistent storage layer where the idempotency utility can store its state - your Lambda functions will need read and write access to it.
+As of now, [Amazon DynamoDB](https://aws.amazon.com/dynamodb/) and [Redis](https://redis.io/) are the supported persistnce layers.
```

Comment:
I'd tie this back to Elasticache - "Redis [...] for instance, as provided by Amazon ElastiCache"

Reply:
@jeromevdl don't agree?

Reply:
nope, it's redis API, which is much broader than elasticache. Elasticache is mentioned in the rest of the document anyway.

Reply:
I think it's worth pointing out the idiomatic thing you need to use in AWS for this, but if it's made clear later 🤷